### DISCUSSION TITLE
Prioritize upload of Saved clips

### DISCUSSION THREAD

**Discussion by Dot-Dash** [marcone/teslausb#884] [Ideas]
Since the driver has deliberately saved those clips, they are probably the interesting ones, and should be uploaded first. It seems like they are being uploaded only after all the Recent clips. Seems like the priority should be Saved, Sentry, then Recents.

**Comment by stayhiddenjt** [comment]
As an alternative, provide the user the ability within the configuration file to specify what priority they would like to use with the default being as suggested here (Saved, Sentry, then Recents).
